src/rdkafka_mock.c

```diff
@@ -469,7 +469,38 @@ rd_kafka_mock_partition_assign_replicas(rd_kafka_mock_partition_t *mpart,
             mpart, mpart->replicas[rd_jitter(0, replica_cnt - 1)]);
 }
 
+/**
+ * @brief Push a partition leader response for passed \p mpart .
+ */
+static void
+rd_kafka_mock_partition_push_leader_response0(rd_kafka_mock_partition_t *mpart,
+                                              int32_t leader_id,
+                                              int32_t leader_epoch) {
+        rd_kafka_mock_partition_leader_t *leader_response;
 
+        leader_response               = rd_calloc(1, sizeof(*leader_response));
+        leader_response->leader_id    = leader_id;
+        leader_response->leader_epoch = leader_epoch;
+        TAILQ_INSERT_TAIL(&mpart->leader_responses, leader_response, link);
+}
+
+/**
+ * @brief First forced partition leader response in passed \p mpart .
```

Comment:
FWIW I don't understand this grammar. Does it mean `\p mpart holds the first forced response from the partition leader`, or I guess more likely `Fetch the first partition leader response into \p mpart`?

Reply:
It's:
`Return the first forced partition leader response in \p mpart , if available`